data/mona/transform.py

```diff
@@ -0,0 +1,211 @@
+import urllib
+import urllib.request
+
+import pandas as pd
+import yaml
+
+DATASET_URL = (
+    "https://huggingface.co/datasets/adamoyoung/mona/resolve/main/data/mona_df.json"
+)
+META_YAML_PATH = "./data/mona/meta.yaml"
```

Comment:
```suggestion
META_YAML_PATH = __file__.replace("transform.py","meta.yaml")
```
This should make it work no matter from which dir you run it.